### ISSUE TITLE
Add aria labels to all fax numbers on VA.gov

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#11466]
## User story

As a Veteran who uses a screen reader, I need fax numbers on VA.gov to be read aloud slowly so that I can understand each number.

## Issue Description
For accessibility reasons we need to add aria labels to all fax numbers on the site. They should not be linked with a href tags because links for faxes will not work on a computer. 

The aria code should be: `<span aria-label="2 1 2. 5 5 5. 1 2 3 4.">212-555-1234</span>`

## Tasks
- [ ] Update all fax numbers on the site so that they have an aria label but are not linked.

Pages with updated fax numbers:

## Acceptance Criteria
- [ ] All fax numbers on site use aria labels and are not linkable
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by RLHecht** [comment]
This issue is blocked until the CMS team can allow `<span>` labels within WYSIWYG fields.

**Comment by FranECross** [comment]
@laflannery Do you know if this is still blocked? Thanks!

**Comment by laflannery** [comment]
For fax numbers, aria-label is not needed because it is not supported on a non-interactive element. [See additional info in this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/60219#issuecomment-1954773645)
